src/pages/article/article.ts

```diff
@@ -0,0 +1,25 @@
+import { Component } from '@angular/core';
+import { IonicPage, NavController, NavParams } from 'ionic-angular';
+
+/**
```

Comment:
this page is out of scope of for this feature

Reply:
done